### PR TITLE
LPS-111561 Responsive for collection display

### PR DESCRIPTION
**Pull request by veroglez**

### CODE REVIEW

**Comment by veroglez** [comment]
ci:test:sf

**Comment by veroglez** [comment]
ci:test:relevant

**Comment by veroglez** [comment]
ci:test:relevant

**Review by p2kmgcl — COMMENTED** [review]
The PR looks nice :)

I just suggested a refactor change in CollectionGeneralPanel, because the component is growing a lot and we might want to split it into smaller ones (or see if we can extract some logic).

Feel free to ignore this or applying the change on a different task :smile:. We can also have a look at it together if you want.

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page-structure/components/item-configuration-panels/CollectionGeneralPanel.js [hunk 1]
```diff
@@ -402,21 +408,59 @@ export function CollectionGeneralPanel({item}) {
...
-													target: {checked},
-												}) =>
-													handleConfigurationChanged({
-														gutters: checked,
-													})
-												}
-											/>
-										</ClayForm.Group>
+										<>
+											<ClayForm.Group small>
```
[the remark this answers is not in the document]
Comment (veroglez) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page-structure/components/item-configuration-panels/CollectionGeneralPanel.js]: It sounds good!

**Comment by Tim-Cao** [comment]
Hi @veroglez, this PR is required to update Poshi codes, Would you mind mentioning us when it is ready?

**Comment by veroglez** [comment]
Hi @Tim-Cao! The PR is ready, could you take a look at the Poshi code?

**Comment by Tim-Cao** [comment]
Sure. I will @veroglez

**Comment by Tim-Cao** [comment]
Just start reviewing

**Comment by Tim-Cao** [comment]
ci:forward

**Comment by Tim-Cao** [comment]
ci:stop

**Comment by Tim-Cao** [comment]
See https://github.com/liferay-echo/liferay-portal/pull/7670

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**